src/flb_gzip.c

```diff
@@ -23,6 +23,7 @@
 #include <fluent-bit/flb_gzip.h>
 #include <fluent-bit/flb_compression.h>
 #include <miniz/miniz.h>
+#include <stdbool.h>
```

Comment:
Could you remove this header dependency and use int type of macros which are FLB_FALSE or FLB_TRUE in this source file?